app/src/main/java/com/berniesanders/connect/data/ActionAlertAdapter.java

```diff
@@ -0,0 +1,66 @@
+package com.berniesanders.connect.data;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import com.berniesanders.connect.R;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A simple array adapter for Action Alerts
+ *
+ * Created by John on 2/21/16.
+ */
+public class ActionAlertAdapter extends BaseAdapter {
+
+    private Context mContext;
+    List<ActionAlert> alerts;
+    private static LayoutInflater inflater = null;
```

Comment:
shouldnt be static.

all of these member variablss look like they can, and therefore should, be final.